vaadin-checkbox-flow-parent/vaadin-checkbox-flow/src/main/java/com/vaadin/flow/component/checkbox/CheckboxGroup.java

```diff
@@ -78,7 +83,7 @@
         MultiSelect<CheckboxGroup<T>, T>,
         HasListDataView<T, CheckboxGroupListDataView<T>>,
         HasDataView<T, Void, CheckboxGroupDataView<T>>, HasHelper, HasLabel,
-        HasTooltip, HasValidator<T> {
+        HasTooltip, HasValidator<Set<T>>, HasClientValidation {
```

Comment:
The value of a `CheckboxGroup` is a `Set<T>`. None of the interface methods had been implemented so far, so this required no further changes.